control-plane/pkg/reconciler/base/receiver_condition_set.go

```diff
@@ -39,14 +39,21 @@ const (
 	ConditionConfigParsed       apis.ConditionType = "ConfigParsed"
 )
 
-var ConditionSet = apis.NewLivingConditionSet(
+var IngressConditionSet = apis.NewLivingConditionSet(
 	ConditionAddressable,
 	ConditionDataPlaneAvailable,
 	ConditionTopicReady,
 	ConditionConfigMapUpdated,
 	ConditionConfigParsed,
 )
 
+var EgressConditionSet = apis.NewLivingConditionSet(
+	ConditionDataPlaneAvailable,
+	ConditionTopicReady,
+	ConditionConfigMapUpdated,
+	ConditionConfigParsed,
+)
+
```

Comment:
would it make sense to just separate this into its own PR, before adding the kafkasource controller and reconciler?

Reply:
I'll open a separate PR